articles/storage/files/storage-troubleshooting-files-performance.md

```diff
@@ -23,7 +23,7 @@ To confirm if your share is being throttled, you can leverage Azure Metrics in t
 
 1. Sign in to the [Azure portal](https://portal.azure.com).
 
-1. Select **All services** and then search for **Metrics**.
+1. Select **Storage account** name and then search for **Monitoring** section.
 
```

Comment:
I will suggest rewriting step 1 through 5 as steps 1-3.

1. Go to your *storage account* in the Azure portal.
2. Select *Metrics* under Monitoring section.
3. Select  *File* as the metric namespace for your storage account scope.